### PR TITLE
NETOBSERV-1298: Add netobserv operator changes to support dup list

### PR DESCRIPTION
**Pull request by msherif1234**
## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

**Comment by jpinsonneau** [comment]
- Updated the frontend columns to a single one for a simpler UI
- Also sending mark & merge agent parameter to frontend configmap to adapt accordingly
see https://github.com/netobserv/network-observability-operator/pull/501/commits/b66c9d6df08cd9c35b237eff5405befe276f0991

File: controllers/ebpf/agent_controller.go [hunk 1]
```diff
@@ -71,6 +69,13 @@ const (
 	bpfNetNSMountPath  = "/var/run/netns"
 )
 
+const (
+	EnvDedupeJustMark     = "DEDUPER_JUST_MARK"
+	EnvDedupeMerge        = "DEDUPER_MERGE"
+	DedupeJustMarkDefault = "true"
+	DedupeMergeDefault    = "false"
+)
+
```
Comment (msherif1234) [inline controllers/ebpf/agent_controller.go:78]: those const are used only locally no need for CAPs

Comment (jpinsonneau) [reply controllers/ebpf/agent_controller.go:78]: I'm actually using these in `consoleplugin_objects.go` so I need to expose them.

Would you prefer to move these to `constants.go` ?

Comment (msherif1234) [reply controllers/ebpf/agent_controller.go:78]: oh sorry my mistake that should be fine then
/lgtm

**Pull request merged**